### PR TITLE
Update README.md to add windows docs

### PR DESCRIPTION
**Pull request by mondus**
Adds details of

- [x] How to install Intel Fortran in Windows
- [x] Details on obtaining the `CMAKE_PREFIX_PATH`
- [x] Cross platform build and install commands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,27 +85,35 @@ To build and install the library:
...
+		  
+4. Make and install the code to the chosen location with either:
+    ```
+    cmake --build .
     ```
-    make
-    make install
+	or (to make and build)
+	```
+    cmake --build . --target INSTALL
```
Comment (jatkinson1000) [inline README.md:109]: Does the former just make/build and the latter makes/builds and then installs?
Is `INSTALL` a keyword for CMake - I have only ever see it used lower case - https://cmake.org/cmake/help/latest/manual/cmake.1.html#build-a-project

**Review by jatkinson1000 — CHANGES_REQUESTED** [review]
Thanks @mondus a couple of small tweaks and a question about the install command.
I've forwarded it to the user and will see how he gets on - using `cmake --build` rather than `make` may be important!

File: README.md (hunk 1 above)
Comment (jatkinson1000) [reply README.md:109]: On reading looks like `install` is for makefile generators, whilst `INSTALL` is for GUI based systems: https://cmake.org/cmake/help/book/mastering-cmake/chapter/Install.html

Will update accordingly and then merge when appropriate.

**Comment by jatkinson1000** [comment]
This is WIP with #50 
Once that is resolved with the user we will update here accordingly and then merging will close #50

**Comment by jatkinson1000** [comment]
I think this is good to go now (correspondent from #50 has gone quiet, but it works for us).
Since I have done a lot of this PR I will let @TomMelt do a final review for approval.

File: README.md [hunk 2]
```diff
@@ -197,18 +229,18 @@ against it.
...
 When compiling the final executable add the following link flag:
-```
+```makefile
 LDFLAGS += -L<path/to/install/location>/lib64 -lftorch
 ```
 
 You may also need to add the location of the `.so` files to your `LD_LIBRARY_PATH`
 unless installing in a default location:
-```
+```makefile
```
Comment (TomMelt) [inline README.md:243]: I think this should technically be "```bash"

**Review by TomMelt — DISMISSED** [review]
looks good to me. Just a couple of questions

File: examples/2_ResNet18/README.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ This can be done using the included `CMakeLists.txt` as follows:
 mkdir build
 cd build
 cmake .. -DFTorch_DIR=<path/to/your/installation/of/library/>lib/cmake/ -DCMAKE_BUILD_TYPE=Release
-make
+cmake --build .
```
[the remark this answers is not in the document]
Comment (jatkinson1000) [reply examples/2_ResNet18/README.md:74]: Yes.

File: README.md (hunk 2 above)
Comment (jatkinson1000) [reply README.md]: Done

**Comment by TomMelt** [comment]
just added a note about loading intel compilers. If @mondus is happy then I think we can merge it in.

**Comment by jatkinson1000** [comment]
> just added a note about loading intel compilers. If @mondus is happy then I think we can merge it in.

@TomMelt Would it be worth extending that by telling users that this can usually be done by running `C:\Program Files (x86)\Intel\oneAPI\setvars` from Cmd?

File: README.md [hunk 4]
```diff
@@ -46,64 +46,89 @@ To install the library requires the following to be installed on the system:
...
+3. Create a `build` directory and execute cmake from within it using the relevant options (see below):  
+    ```bash
     mkdir build
     cd build
     cmake .. -DCMAKE_BUILD_TYPE=Release
     ```
-    It is likely that you will need to provide at least the `CMAKE_PREFIX_PATH` flag.  
-    The following CMake flags are available and can be passed as arguments through `-D<Option>=<Value>`:
+    If building on Windows you will need to add `-G "NMake Makefiles"` to the `cmake` command. Also you will need to load the
+    intel fortran compilers using `setvars.bat` which is found in the Intel compiler install directory.
```
Comment (mondus) [inline README.md:85]:
```suggestion
    intel fortran compilers using `setvars.bat` ([see Intel docs](https://www.intel.com/content/www/us/en/docs/oneapi/programming-guide/2023-2/use-the-setvars-script-with-windows.html)) which is found in the Intel compiler install directory.
```

**Review by mondus — COMMENTED** [review]
Tested in Windows. All good. Happy to approve. One minor suggestion comment to add a link to Intel guide on setvars.

**Comment by jatkinson1000** [comment]
Thanks @mondus, that also addresses my comment above but in an even safer/better way than suggested, thanks.
I'll approve and merge now (@mondus has confirmed offline that he is happy with the PR).

**Review by jatkinson1000 — APPROVED** [review]
Thanks all who contributed here, I'm now happy to go ahead so will merge in a moment.

**Pull request merged**